src/AccountingToken.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.28;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+// TODO maybe permissioned transfers
+contract AccountingToken is ERC20 {
```

Comment:
we' re currently defaulting to upgradeable contracts for all deployments, to keep things adaptable if bussiness requirements change